R/geom-.r

```diff
@@ -321,6 +321,15 @@ Geom <- gganimintproto("Geom",
     if(zero.size && has.no.fill){
       warning(sprintf("geom_%s with size=0 will be invisible",g$geom))
     }
+
+    ## raise warning for using _off params without clickSelects
+    has.off <- any(names(g$params) %like% "_off")
+    has.no.cs <- !any(is.cs)
+    if(has.no.cs && has.off){
+      off.vec <- grep( "_off$", names(g$params), value = TRUE)
+      warning(sprintf("geom_%s has %s which is not used because this geom has no clickSelects; please specify clickSelects or remove %s",
```

Comment:
instead of just g$geom / geom_point, can you please display g$classed or layer_name? (which should be something like geom1_point_scatter, that gives more information -- geom number and parent plot name)

Reply:
Yes, on it now